### DISCUSSION TITLE
Plans for local file exporter for .net

### DISCUSSION THREAD

**Discussion by Jebarson** [open-telemetry/opentelemetry-dotnet#5984] [Q&A] [answered]
Are there any plans for releasing a local file exporter for .net? 

I understand that the collector exists but i am looking for a simple solution that I could leverage for windows applications (not console) to export to local files. I am in fact looking for a unstructured one and not necessarily a json formatted.

I am happy to contribute as long as the owners are happy to incorporate. If not, I would fork it build one.

**Comment by cijothomas** [comment, marked as answer]
Once this is merged (https://github.com/open-telemetry/opentelemetry-specification/pull/4183), we could add a local file exporter. Most likely, the existing ConsoleExporter will be left as-is, so this can be a new component, perhaps even hosted in Contrib repo. I don't think there'll be any harm is working on one right away in the contrib repo.
